bakerydemo/base/fixtures/bakerydemo.json

```diff
@@ -3531,6 +3852,54 @@
       "theme": "system"
     }
   },
+  {
+    "model": "wagtailusers.userprofile",
+    "pk": 2,
+    "fields": {
+      "user": ["editor"],
+      "submitted_notifications": true,
+      "approved_notifications": true,
+      "rejected_notifications": true,
+      "updated_comments_notifications": true,
+      "preferred_language": "",
+      "current_time_zone": "",
+      "avatar": "",
+      "dismissibles": {},
+      "theme": "system"
+    }
+  },
+  {
+    "model": "wagtailusers.userprofile",
+    "pk": 3,
+    "fields": {
+      "user": ["german"],
+      "submitted_notifications": true,
+      "approved_notifications": true,
+      "rejected_notifications": true,
+      "updated_comments_notifications": true,
+      "preferred_language": "de",
+      "current_time_zone": "Europe/Berlin",
+      "avatar": "",
+      "dismissibles": {},
+      "theme": "system"
+    }
+  },
+  {
+    "model": "wagtailusers.userprofile",
+    "pk": 4,
+    "fields": {
+      "user": ["arabic"],
+      "submitted_notifications": true,
+      "approved_notifications": true,
+      "rejected_notifications": true,
+      "updated_comments_notifications": true,
+      "preferred_language": "ar",
+      "current_time_zone": "Asia/Beirut",
+      "avatar": "",
+      "dismissibles": {},
+      "theme": "system"
+    }
+  },
```

Comment:
While working on some extra demo data I noticed the user profile for the moderator user is missing here. The inactive user profile is missing too.

Is this intentional?

Reply:
Hmm, `inactive` most likely yes but moderator no. I’ll fix this and merge.